compensating-transactions/mongodb-simple/pom.xml

```diff
@@ -115,7 +83,7 @@
             <exclusions>
                 <exclusion>
                     <groupId>org.jboss.logging</groupId>
-                    <artifactId>jboss-logging-spi</artifactId>
+                    <artifactId>jboss-logging-spi-</artifactId>
```

Comment:
typo

Reply:
That was hard to spot! Thanks @marcosgopen